openedx/core/djangoapps/credit/models.py

```diff
@@ -18,6 +18,16 @@
 log = logging.getLogger(__name__)
 
 
+class CreditProvider(TimeStampedModel):
+    """
+    This model represents an institution that can grant credit for a course.
+    Each provider is identified by unique ID (e.g., 'ASU').
+    """
+
+    provider_id = models.CharField(max_length=255, db_index=True, unique=True)
```

Comment:
Minor: This name is too close to `id` which is already a field on the model. Consider changing to `code` or something that cannot be confused with the existing `id` field.